lnwallet/fee_estimator_test.go

```diff
@@ -144,3 +160,71 @@ func TestSparseConfFeeSource(t *testing.T) {
 		t.Fatalf("expected ParseResponse to fail")
 	}
 }
+
+func TestWebAPIFeeEstimator(t *testing.T) {
+	t.Parallel()
+
+	invalidBlockTarget := uint32(1)
+	blockTargetNoFee := uint32(2)
+	blockTargetSmallFee := uint32(10)
+	validBlockTarget := uint32(20)
+	testFees := map[uint32]uint32{
+		invalidBlockTarget:  12345,
+		blockTargetSmallFee: 42,
+		validBlockTarget:    54321,
+	}
+	feeSource := mockSparseConfFeeSource{
+		url:  "https://www.github.com",
+		fees: testFees,
+	}
+	estimator := lnwallet.NewWebAPIFeeEstimator(feeSource, 10)
+
+	// Test that requesting a fee for a too-low block target will error.
```

Comment:
Many of these assertions below can be converted to use [sub-tests](https://golang.org/pkg/testing/#hdr-Subtests_and_Sub_benchmarks).

Reply:
fixed